engine/pom.xml

```diff
@@ -272,7 +272,6 @@
       <groupId>org.eclipse.jetty</groupId>
       <artifactId>jetty-websocket</artifactId>
       <version>${jetty.version}</version>
-      <scope>test</scope>
```

Comment:
This is because previously, there was a unit test that is based on the websocket in Apex.  But now since the websocket servlet is moved to Apex, the dependency scope becomes not just test, since the websocket servlet code depends on the websocket library.